test-suite/handwritten-src/java/android/os/Parcel.java

```diff
@@ -0,0 +1,155 @@
+package android.os;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+
+public final class Parcel {
+
+    private FileOutputStream mOutFile;
+    private FileInputStream mInFile;
+    private ObjectOutputStream mOut;
+    private ObjectInputStream mIn;
+
+    public Parcel() {
+        final String fname = "parcel.tmp";
```

Comment:
This will create this temp-file somewhere (path unspecified) and never clean it up?  I'd prefer to see the test suite explicitly create the file in a known location and clean it up after the tests are done.